report/src/introduction/agile_approach.tex

```diff
@@ -1,29 +1,27 @@
 \section{Implementing Agile Project Management with Scrum}
 Developing a software product for a business can be a complex process.
-Incremental agile approaches have become popular to address this complexity, as they allow for faster development and adaptation to changes.\cite{sommervilleSoftwareEngineering2016}
-In incremental agile approaches, a system is developed as a series of increments, each adding functionality to the system.
-Between these increments, one gathers feedback from users to prioritize functionality for the next increment.
+Incremental agile approaches have become popular to address this complexity, as it allows for fast adaptation to changes in user needs. \cite{sommervilleSoftwareEngineering2016}
+In incremental-agile approaches, system development is considered a series of increments, each adding functionality to the system.
+Between these increments, one gathers feedback from stakeholders to prioritize functionality for the next increment.
 Therefore, each increment adds value to the user.
-By continuously getting feedback and working with these increments, the development process becomes cheaper and faster as a result of less time spent reworking the system.\cite{sommervilleSoftwareEngineering2016}
-This section will discuss how we are using an agile approach for our project, focusing specifically on the Scrum framework.
+By continuously getting feedback and working with these increments, the development process becomes cheaper and faster as less time is spent adapting to changes.\cite{sommervilleSoftwareEngineering2016}
+This section will discuss how an agile approach has been utilized in this project. 
 
-\subsection*{Scrum Framework}
-Agile is a set of values, and does not present practical activities which a process model would consist of\cite{sutherlandScrumArtDoing2014}.
-Therefore, the team has decided on Scrum\cite{scrumdotorg}, which is an agile framework that puts the values to practice.
-This decision was made due to prior experience with using the framework amongst team members.
+\subsection{Scrum Framework}
```

Comment:
```suggestion
\subsection*{Scrum Framework}
```

Reply:
Or the ToC looks like this:
![image](https://user-images.githubusercontent.com/29108628/226850103-6014f439-f2dd-4970-bfe1-ac54a2429039.png)